cypress/integration/plugins/dashboards-assistant/conversation_history_spec.js

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { BASE_PATH } from '../../../utils/constants';
+
+const setLocalStorageItem = (key, value) => {
+  const oldValue = localStorage.getItem(key);
+  localStorage.setItem(key, value);
+  return () => {
+    if (oldValue === null) {
+      localStorage.removeItem(key);
+    } else {
+      localStorage.setItem(key, oldValue);
+    }
+  };
+};
```

Comment:
extract into util class, so we can reuse for other test caes?

Reply:
Sure, can be move to assistant utils folder.